DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs

```diff
@@ -60,16 +61,18 @@ public string Content
         public bool IsTTS { get; set; } = false;
 
         /// <summary>
-        /// Gets the Allowed Mentions for the message to be sent. 
+        /// Gets the Allowed Mentions for the message to be sent.
         /// </summary>
         public List<IMention> Mentions { get; private set; } = null;
 
         /// <summary>
         /// Gets the Files to be sent in the Message.
         /// </summary>
         public IReadOnlyCollection<DiscordMessageFile> Files => this._files;
+        public IReadOnlyList<DiscordActionRowComponent> Components => this._components;
+        internal readonly List<DiscordActionRowComponent> _components = new(5);
 
-        internal List<DiscordMessageFile> _files = new();
+        internal readonly List<DiscordMessageFile> _files = new();
```

Comment:
Put the components members after `_files` and not in the middle. Also, document `Components`.

Reply:
Another tired mistake. Sorry.